indico/http/client.py

```diff
@@ -186,3 +194,165 @@ def _make_request(
                 error=error, code=response.status_code, extras=extras
             )
         return content
+
+
+class AIOHTTPClient(HTTPClient):
+    """
+    Beta client with a minimal set of features that can execute
+    requests using the aiohttp library
+    """
+
+    def __init__(self, config: Optional[IndicoConfig] = None):
+        """
+        Config options specific to aiohttp
+        unsafe - allows interacting with IP urls
+        """
+        self.config = config or IndicoConfig()
+        self.base_url = f"{self.config.protocol}://{self.config.host}"
+        unsafe = config.verify_ssl
```

Comment:
Would we want to tie this to verify_ssl? Unsafe seems to allow cookies from IP addresses, but that would be slightly different behavior from allowing self signed ssl.

Reply:
great catch - looks like I use this to make requests anyway, not sure why I added this unless it was my dumb attempt at the cookie stuff - will remove!

Reply:
All good! Nothing dumb about it, just part of the process 😅